lms/envs/test.py

```diff
@@ -484,6 +484,9 @@
 # Enable teams feature for tests.
 FEATURES['ENABLE_TEAMS'] = True
 
+# Enable indexing teams for search
+FEATURES['ENABLE_TEAMS_INDEX'] = True
```

Comment:
One option would be to push a branch to the configuration repo which sets the flag to True, then build all sandboxes for teams with this branch.